.claude/commands/sh-release-note.md

```
@@ -0,0 +1,95 @@
+- **File exists** → follow the UPDATE path (steps 3b, 4b, skip 5a)
+
+---
+
+### CREATE path
+
+#### Step 3a — Create the changelog detail page
+
+Create `docs/self-hosted/changelog/{slug}.mdx`.
+
+Frontmatter:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)